ui/raidboss/popup-text.ts

```diff
@@ -1089,7 +1089,7 @@ export class PopupText {
   // Set defaults for triggerHelper object (anything that won't change based on
   // other trigger functions running)
   _onTriggerInternalHelperDefaults(triggerHelper: TriggerHelper): void {
-    if (triggerHelper.triggerAutoConfig) {
+    {
```

Comment:
This scoping feels a bit bad to be honest, maybe the const variables should just be renamed?